### PR TITLE
SnakySnake-TheBunOne

### PR DESCRIPTION
**Pull request by TheBunOne**
I came into this project thinking that it would be relatively simple and I would be able to create a masterpiece fit for any museum. _Boy was I wrong._
I would like to say that coming into this project I had absolutely no knowledge of java. The only code that I had experience with was python, and suffice to say that my knowledge was/is not very great as I am completely self taught (and take it from me, I'm a horrible teacher). 
After many hours of messing around, and trying to figure out what the heck all of the things meant/how the operations work,(does anyone really know what a non-uniform rational b-spline is?) I decided that I would have to give up my "delusions of grandeur " as my friend calls them. I settled for an easy pattern of a snake that uses randomness to create curves, such as those shown. 

Snap1
Snap2
Snap3

However hard I make it out to be, I really did have a lot of fun doing this, and also gained more useful coding knowledge. 
I hope my piece is not too woefully lacking....
-TheBunOne :)

### CODE REVIEW

**Comment by leomcelroy** [comment]
Good project management on scaling back to get something done. You'll work up to the grand piece.

I think it earns a blot. You just need to add `.png` to the snapshot title in the metadata.

File: art/Index.js.js
Comment (TheBunOne) [inline art/Index.js.js:1]: Wrong one, sorry!

File: art/Index.js
Comment (TheBunOne) [inline art/Index.js:1]: This is the updated file

**Comment by TheBunOne** [comment]
I updated the metadata, thank you SO much!

**Comment by leomcelroy** [comment]
You've got errant files added outside of your folder.

**Comment by leomcelroy** [comment]
I'm going to close this for housekeeping. Feel free to resubmit in the future.

**Pull request closed without merging**